setup.py

```diff
@@ -148,6 +163,7 @@ def check_cmake_in_path():
         language="c++",
         define_macros=[
             ("AUDIO_PLAY_NONE", None),
+            ("NPY_NO_DEPRECATED_API", "NPY_1_7_API_VERSION"),
```

Comment:
From Cython 3.0 onwards, defining `NPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION` will prevent the deprecated Numpy C API from being used.
https://cython.readthedocs.io/en/3.0.x/src/userguide/migrating_to_cy30.html#numpy-c-api
https://cython.readthedocs.io/en/latest/src/userguide/numpy_tutorial.html#compilation-using-setuptools

It is also required for the `pyopenjtalk.openjtalk` module, but it will be omitted in this PR because it will no longer be necessary in #87.